scripts/ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+setupNvm() {
+    export NVM_DIR="$HOME/.nvm"
+    [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+
+    nvm install
+    nvm use
+}
+
+globalJsDependencies() {
+    npm install -g yarn
+}
+
+yarnSetup() {
+    yarn install --force
```

Comment:
This doesn't yield a deterministic build as the lockfile get's completely ignored - `yarn install --frozen-lockfile` is preferred. From https://classic.yarnpkg.com/en/docs/cli/install/:

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass `--frozen-lockfile` flag.

Reply:
Good advice, thanks @akash1810!